test/e2e/user_management_test.go

```diff
@@ -256,6 +256,9 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 
 		// The non-excluded user should be deactivated
 		err = hostAwait.WaitUntilMasterUserRecordAndSpaceBindingsDeleted(t, murMember1.Name)
+		bytes, err2 := wait.StringifyObject(userSignupMember1)
+		require.NoError(t, err2)
+		t.Logf("### DEBUG usersignup: %s", string(bytes))
```

Comment:
most likely some left-over from troubleshooting?

Reply:
not quite left over, i'm actively trying to diagnose the cause of the current test failures.